internal/database/export.go

```diff
@@ -156,17 +159,17 @@ func (db *DB) AddExportBatches(ctx context.Context, batches []*model.ExportBatch
 		_, err := tx.Prepare(ctx, stmtName, `
 			INSERT INTO
 				ExportBatch
-				(config_id, bucket_name, filename_root, start_timestamp, end_timestamp, region, status, signing_key)
+				(config_id, bucket_name, filename_root, start_timestamp, end_timestamp, region, status, signing_key, signing_key_id, signing_key_version, app_package_name, bundle_id)
```

Comment:
Is there a reason to store the key version and key id separately? I don't see them being used separately?

Reply:
I'm going to need to extract this information out to a separate table anyway - see @gurayAlsac comment in this PR.